docs/tasks/administer-cluster/kubeadm-upgrade-1-7.md

```diff
@@ -0,0 +1,56 @@
+---
+assignees:
+- pipejakob
+title: Upgrading kubeadm clusters to 1.7
+redirect_from:
+- "/docs/admin/kubeadm-upgrade-1-7/"
+- "/docs/admin/kubeadm-upgrade-1-7.html"
+---
+
+* TOC
+{:toc}
+
+This guide is for upgrading kubeadm clusters from version 1.6.x to 1.7.x, or
+other 1.6.x releases. Upgrades are not supported before the 1.6.0 release (when
+kubeadm became Beta).
+
+It is a work-in-progress (not intended for merging yet) and targeted at other
```

Comment:
I put the do-not-merge label on so you can remove this, plus the WIP in title also denotes.